doc/releases/release-notes-4.0.rst

```diff
@@ -342,6 +448,11 @@ Drivers and Sensors
     :dtcompatible:`renesas,ra-cgc-pclk-block`, :dtcompatible:`renesas,ra-cgc-pll`,
     :dtcompatible:`renesas,ra-cgc-external-clock`, :dtcompatible:`renesas,ra-cgc-subclk`,
     :dtcompatible:`renesas,ra-cgc-pll-out`)
+  * Introduced ARM's System Control and Management Interface (SCMI), clock management
```

Comment:
introduction of SCMI in general probably deserves its own entry (under "Architectures")? @LaurentiuM1234

Reply:
hmm, or maybe have it under "Libraries/Subsystems" section as its own subsection or add a "Firmware" section under "Libraries/Subsystems" and have it there? but then again the main idea is to inform people that Zephyr has SCMI support so I'm fine with having it anywhere as long as it makes sense.

Reply:
can drop this. Submitted https://github.com/zephyrproject-rtos/zephyr/pull/81330 for the SCMI notes.